browser/app/js/components/Login.js

```diff
@@ -14,115 +14,118 @@
  * limitations under the License.
  */
 
-import React from 'react'
-import classNames from 'classnames'
-import logo from '../../img/logo.svg'
-import Alert from 'react-bootstrap/lib/Alert'
-import * as actions from '../actions'
-import InputGroup from '../components/InputGroup'
+import React from "react"
+import { connect } from "react-redux"
+import classNames from "classnames"
+import logo from "../../img/logo.svg"
```

Comment:
Can you use the new logo here (please check the attachement)? 
[logo.zip](https://github.com/minio/minio/files/1682433/logo.zip).

Reply:
yes, will do

Reply:
@rushenn @kmkanagaraj Do we need to use the new logo for the favicon as well?